data/hai missions.txt

```diff
@@ -1061,3 +1061,366 @@ mission "Hai Honeymoon"
+				accept
+	on complete
+		conversation
+			`The Hai, as usual, mysteriously and efficiently deliver your message from Ambassador Sayari, and a local prefect appears within minutes to inquire if you've encountered a friendly race with hull repair technology.`
+			`The prefect continues to tell you how the Solifuge was not entirely a surprise to the Hai government, but that it posed a threat to the Hai public mindset that couldn't be ignored.`
+
+mission "Strider: Have language, will travel"
+	description "Take Ambassador Yashili and her delegation to <destination>."
+	name "Have language, will travel"
+	landing
+	source "Hai-home"
+	destination "Greenwater"
+	passengers 3
+	blocked "The Hai wish to send a delegation to a species that has hull repair technology, and will require 3 bunks for the trip."
+	to offer
+		has "Strider: View of the Pond: done"
+	on offer
+		log `The Hai are working on a Medium Warship that carries drones, but need help contacting a species with hull repair technology to make it viable.`
+		conversation
+			`"A small, short range drone carrier and combat drone solution is being worked on, but initial simulations predict that actual escort casualties would go up unless significant hull repair technology could be implemented to fix docked drones that would otherwise weigh down the agile warship" the prefect explains.`
+			branch neither
+				not "remnant blood test pure"
+				not "Coalition: First Contact: offered"
+			`The delegation will need to stop at <destination> to pick up translation devices they can use to communicate with other species.`
+			branch coalition
+				has "Coalition: First Contact: offered"
+				not "remnant blood test pure"
+			branch remnants
+				has "remnant blood test pure"
+				not "Coalition: First Contact: offered"
+			branch both
+				has "remnant blood test pure"
+				has "Coalition: First Contact: offered"
+			label coalition
+			`You consider your encounters with the Coalition and their repair outfits.`
+			choice
+				`Mention the Coalition, and offer to take a delegation to the Ring of Friendship.`
+					goto ring
+				`The Hai are not ready to learn of the Coalition's existence yet.`
+					defer
+			label remnants
+			`You consider your encounters with the Remnants and their self-repairing hulls.`
+			choice
+				`Mention the Remnants, and offer to take a delegation to Aventine.`
+					goto aventine
+				`The Hai are not ready to learn of the Remnants' existence yet.`
+					defer
+			label both
+			`You consider that both the Remnants and the Coalition have technology that repairs hulls.`
+			choice
+				`Mention the Remnants, and offer to take a delegation to Aventine.`
+					goto aventine
+				`Mention the Coalition, and offer to take a delegation to the Ring of Friendship.`
+					goto ring
+				`The Hai are not ready to learn of hull repair technology yet.`
+					defer
```

Comment:
Instead of straight up deferring for the final choice in these three instances, have these `goto neither`.

Reply:
way to make the most use of my written words - thanks!